hadoop-common-project/hadoop-common/pom.xml

```diff
@@ -391,11 +391,19 @@
     <dependency>
       <groupId>com.squareup.okhttp3</groupId>
       <artifactId>mockwebserver</artifactId>
+      <version>${okhttp3.version}</version>
```

Comment:
prefer hadoop-project retains the declaration, but now with all the excludes. the modules should get the exclusions automatically, shouldn't they?